pyquil/_parser/PyQuilListener.py

```diff
@@ -406,52 +406,52 @@ def exitDefWaveform(self, ctx:QuilParser.DefWaveformContext):
 
     def exitPulse(self, ctx:QuilParser.PulseContext):
         qubits = list(map(_formal_qubit, ctx.formalQubit()))
-        frame = ctx.frame().getText()
+        frame = ctx.frame().getText()[1:-1]
```

Comment:
Does `getText()` return a string? If so, there is always the `.strip('"\'')` method.

Reply:
It seems that this has changed, but this pattern is what's used up in `exitPragma` as well.